data/json/items/gun/monster_gun.json

```diff
@@ -65,7 +65,9 @@
     "material": [ "stone" ],
     "flags": [ "PRIMITIVE_RANGED_WEAPON", "NEVER_JAMS", "NONCONDUCTIVE", "NO_REPAIR", "WATERPROOF_GUN", "NO_SALVAGE", "NO_UNLOAD" ],
     "skill": "throw",
+    "ammo": [ "rock" ],
     "ammo_effects": [ "NO_PENETRATE_OBSTACLES" ],
+    "clip_size": 1,
```

Comment:
Make that six rocks, please.

Reply:
Are you sure? It's not ammo count, and a `slingshot` has the same `clip_size` of `1`. The ammo count is defined as `starting_ammo` in monster definition and is set to `6`.

Reply:
Sorry I was confused. I’ll figure out how to change my review. Thanks for fixing this.